docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3"
+services:
+  app:
+    build:
+      context: ./docker
+    container_name: focalboard
+    expose: 
+      - 8000
+    environment:
+      - VIRTUAL_HOST=focalboard.local
+      - VIRTUAL_PORT=8000
+  
+  proxy:
```

Comment:
the proxy complicates the example tbh, just expose focalboard on some http port :-)

Reply:
I can do that, would you be open to a ```docker-compose.yml``` and ```docker-compose-nginx.yml``` setup  so that both examples are available?

Reply:
added additional docker-compose file